<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBarsDialog</class>
 <widget class="QDialog" name="EditBarsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Bars</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="selectedBarsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Selected Bars</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QTextEdit" name="selectedBarsTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QGroupBox" name="endJointsGroupBox">
     <property name="title">
      <string>End Joints</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fromJointLabel">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ComboBox" name="fromJointComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="toJointLabel">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBox" name="toJointComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="areaLabel">
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="areaDoubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="modulusLabel">
     <property name="text">
      <string>Modulus</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="modulusDoubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="factorLabel">
     <property name="text">
      <string>Factor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="factorDoubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="unitWeightLabel">
     <property name="text">
      <string>Unit Weight</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="unitWeightDoubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>59</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="editBarsPushButton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="icon">
      <iconset resource="../../rsc.qrc">
       <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
